pyproject.toml

```diff
@@ -30,15 +29,15 @@ SQLAlchemy-Utils = "^0.38.2"
 starlette = "^0.27.0"
 tenacity = "^8.0.1"
 uvicorn = { extras = ["standard"], version = "^0.20.0" }
-alembic-utils = "^0.8.2"
 botocore = "^1.34.19"
+db-client = {git = "https://github.com/climatepolicyradar/navigator-db-client.git", tag = "main-dc7d0417"}
```

Comment:
Any reason to not use the same semvar release format we tend to use elsewhere here for the tag?